docs/photoniq/p3/manage-p3-cache.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 40
+title: Manage P3 Cache
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+You can manage the PhotonIQ Performance Proxy (P3) cache in the PhotonIQ Performance Proxy dashboard.
+
+## View Purge Cache Requests
+
+Perform the following steps to view P3 cache purge requests:
+
+1. Log in to your PhotonIQ Performance Proxy dashboard.
+2. Click **Cache**.
+
+    The Purge Cache Requests table provides the following fields:
+
+    - **URL** - The URL of the cache being purged.
```

Comment:
I need some details here. What are the requirements for the URL? Origin URL? Part of the Origin? What does the user need to know about this?